pandas/_libs/tslibs/np_datetime.pyx

```diff
@@ -100,6 +95,28 @@ def py_get_unit_from_dtype(dtype):
 # Comparison
 
 
+cdef bint cmp_dtstructs(
```

Comment:
So this will be tested in a follow up PR when integrated into `Timestamp` & `Timedelta`?

Reply:
yes